File: tests/csharp/TestInstantiation.cs
using System;
using KeyNetra.Client;

namespace KeyNetra.Tests
{
    class Program
    {
        static void Main(string[] args)
        {
            try {
                var client = new KeyNetraClient("http://localhost:8080", "test-key");
                if (client != null) {
                    Console.WriteLine("C# SDK: Client instantiated successfully.");
                }
            } catch (Exception e) {
                Console.Error.WriteLine("C# SDK: Error: " + e.Message);
                Environment.Exit(1);
            }
        }
    }
}

File: tests/java/TestInstantiation.java
import io.keynetra.client.KeyNetraClient;

public class TestInstantiation {
    public static void main(String[] args) {
        try {
            KeyNetraClient client = new KeyNetraClient("http://localhost:8080", "test-key");
            if (client != null) {
                System.out.println("Java SDK: Client instantiated successfully.");
            }
        } catch (Exception e) {
            System.err.println("Java SDK: Error: " + e.getMessage());
            System.exit(1);
        }
    }
}
